src/tipc_address.cpp

```diff
@@ -93,6 +114,25 @@ int zmq::tipc_address_t::resolve (const char *name)
     address.addr.nameseq.upper = upper;
     address.scope = TIPC_ZONE_SCOPE;
     return 0;
+portid:
```

Comment:
isn't this much simpler in the if (res == 0) block above? There's no reuse of this block